hack/integration-test.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 Red Hat, Inc.
+#
+
+set -e
+
+# INTEG_TEST_IMAGE=${INTEG_TEST_IMAGE:-"docker.io/library/golang:1.19.5-bullseye"}
+INTEG_TEST_IMAGE=${INTEG_TEST_IMAGE:-"quay.io/kubevirt/builder:2212180911-8818abcfa"}
```

Comment:
Please consider using a more lightweight image.
Using this image + tag combination will cause download of a large (and old) image.

Reply:
This is the builder image which is used for many other operations, it makes sense to re-use it for this task as well instead of creating a new one from scratch.